conans/cli/api/subapi/remotes.py

```diff
@@ -35,6 +35,14 @@ def get(self, remote_name):
         app = ConanApp(self.conan_api.cache_folder)
         return app.cache.remotes_registry.read(remote_name)
 
+    @api_method
+    def get_remotes(self, remote_names=None):
+        app = ConanApp(self.conan_api.cache_folder)
+        if remote_names:
+            return [self.get(remote_name) for remote_name in remote_names]
+        elif remote_names is None:  # if we don't pass any remotes we want to retrieve only the enabled ones
+            return app.cache.remotes_registry.list(filter_disabled=True)
```

Comment:
I don't think this code belongs to the remotes API, the `# if we don't pass any remotes we want to retrieve only the enabled ones` is logic from the command. We might just share a function in the `cli/commands/__init__` or so.